### ISSUE TITLE
how to create dataset with label??

### ISSUE THREAD

**Issue by Johnson-yue** [dvschultz/stylegan2-ada#5]
Hi, would you say the detail about how to create dataset with label ?
and finetune processing ???
thank you

**Comment by dvschultz** [comment]
The way i have this set up is to use a main folder with sub folders with each class

mainfolder
|-class0 folder
|-class1 folder
|-class2 folder
|-etc

Then run `python dataset_tool.py create_from_image_folders ./datasets/dataset-name /path/to/mainfolder`

**Comment by dvschultz** [comment]
you can’t finetune from labeled datasets unless you have a pretrained model with the same number of classes.

**Comment by Johnson-yue** [comment]
@dvschultz yes， I know it , So my method is only trained conditional layers，and freezed unconditional layers train
